src/material-experimental/theming/_format-tokens.scss

```diff
@@ -0,0 +1,5 @@
+// DO NOT REMOVE
+// This function is used internally.
+@function private-internal-format($systems) {
```

Comment:
Maybe something like `private-format-tokens`? `format` is a bit generic on its own.